index.html

```diff
@@ -13,16 +13,16 @@
 </head>
 <body>
   <div id="mobile">
-      <div>
+      <div id="mobile-header">
           <nav>
               <div class="mobile-logo-placeholder"><img src="img/001-My-Logo-img.png" alt="Home"></div>
               <div class="mobile-button-placeholder"><img src="img/NormalButton/Tertiary/Icons/Union.png" alt=""></div>
           </nav>
           <section id="mobile-bg">
               <div id="mobile-headline">
-                  <h1>I’m Mariane
+                  <h1 class="primary-text-title">I’m Mariane
                     Glad to see you!</h1>
-                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <p class="primary-text-paragraph">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
                     <div class="mobile-headline-supporting-text">
                         <h2>LET’S CONNECT</h2>
                     </div>
```

Comment:
It looks really good, just remember to keep the identation in 2 spaces to keep the best HTML & CSS practices and it will be perfect.